core/controllers/cron.py

```diff
@@ -38,6 +38,15 @@
 def require_cron_or_superadmin(handler):
     """Decorator to ensure that the handler is being called by cron or by a
     superadmin of the application.
+
+    Args:
+        handler: A handler instance
```

Comment:
Actually the argument here is a function, so the existing naming is misleading (each of the handlers is actually a class). Can we rename `handler` in this function to `func` throughout? Then the arg is:

    func: The cron function to be decorated.